fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPage.kt

```diff
@@ -215,17 +220,29 @@ private fun LinkText(
         )
     }
 
+    // While just using UrlAnnotation, talkback shows links in a separate dialog in that case and
+    // opens them on the default browser. Since this component allows the caller to define the
+    // onClick behaviour - e.g. to open the link in in-app custom tab, here StringAnnotation is used
+    // and modifier is enabled with Role.Button when screen reader is enabled.
```

Comment:
nit: could this comment be a little clearer e.g. which component?
```suggestion
    // When using UrlAnnotation, talkback shows links in a separate dialog and
    // opens them in the default browser. Since this component allows the caller to define the
    // onClick behaviour - e.g. to open the link in in-app custom tab, here StringAnnotation is used
    // and modifier is enabled with Role.Button when screen reader is enabled.
```